expression-language/5.0/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Jakarta Expression Language 5.0 (under development)"
+date: 2021-04-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta Expression Language defines an expression language for Java applications.
+
+* [Jakarta Expression Language 5.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/5.0.0)
+    * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+
+# Plan
+
+The plan for the release is to tackle the open issues for the API component including:
+* Potential extensions to the API to support Jakarta Faces
+* Support for default methods
+* Potentially removing the dependency on the java.desktop JPMS module
+* Remove deprecated methods from API
+
+# Backwards compatibility
+Backward compatibility issue include:
+* Removal of deprecated methods
+* Removing the java.desktop dependency may create compatibility issues. The extent of those issues will be a factor in deciding whether to proceed with this or not.
```

Comment:
Is there an action expected to remove the `java.desktop` dependency?  I couldn't see any changes on the master branch related to "java.desktop".

For reference from the https://docs.oracle.com/javase/9/docs/api/java.desktop-summary.html description:
```
Defines the AWT and Swing user interface toolkits, plus APIs for 
accessibility, audio, imaging, printing, and JavaBeans.
```

Reply:
https://github.com/eclipse-ee4j/el-ri/issues/47

I've only taken a quick look at it so far. There isn't a simple solution. It needs a wider a discussion in the community to figure out how we want to handle this.